test/python/result/test_result.py

```diff
@@ -133,9 +133,8 @@ def test_marginal_counts_result(self):
         self.assertEqual(marginal_counts(result, [0]).get_counts(1),
                          expected_marginal_counts_2)
 
-    def test_marginal_counts_in_place_result(self):
-        """Test that a Result object containing counts marginalizes correctly
-            when in_place = True.
+    def test_marginal_counts_inplace_true(self):
+        """Test marginal_counts(Result, inplace = True)
         """
```

Comment:
```suggestion
        """Test marginal_counts(Result, inplace=True)"""
```

Reply:
Done. Please have a look at the latest commit @taalexander and @1ucian0. Is there any other changes I need to make?